src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -16,44 +16,44 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9",
+        "php": "^7.1.3",
         "ext-xml": "*",
-        "symfony/cache": "~3.3|~4.0.0",
-        "symfony/class-loader": "~3.2",
-        "symfony/dependency-injection": "~3.3-beta2|~4.0.0",
-        "symfony/config": "~3.3|~4.0.0",
-        "symfony/event-dispatcher": "~3.3|~4.0.0",
-        "symfony/http-foundation": "~3.3|~4.0.0",
-        "symfony/http-kernel": "~3.3|~4.0.0",
+        "symfony/cache": "~3.4|~4.0",
+        "symfony/class-loader": "~3.4|~4.0",
```

Comment:
yeah, but the requirement will be dropped entirely when removing deprecated code anyway (only the BC layer requires it), so it is fine for now